### ISSUE TITLE
Accessing PropTypes via the main React package is deprecated

### ISSUE THREAD

**Issue by hckhanh** [yahoo/react-intl#933]
Hi there,
When I upgrade to the newer version of React (15.5.4). There are some warnings about [PropTypes](https://facebook.github.io/react/docs/typechecking-with-proptypes.html).
Now, Facebook refer to the [prop-types](https://github.com/reactjs/prop-types) package instead of built-in PropTypes in main package.

```
Warning: Accessing PropTypes via the main React package is deprecated. Use the prop-types package from npm instead
```

**Comment by hatchette** [comment]
This is duplicate of #922

**Comment by MichaelDeBoey** [comment]
Closed by #918
